pkg/cluster/spec/spec.go

```diff
@@ -112,6 +112,7 @@ type (
 		Pump           map[string]interface{} `yaml:"pump"`
 		Drainer        map[string]interface{} `yaml:"drainer"`
 		CDC            map[string]interface{} `yaml:"cdc"`
+		TiKVCDC        map[string]interface{} `yaml:"tikv-cdc"`
```

Comment:
Better use `tikv_cdc` to keep the same naming format with other components.

Reply:
The reason why I chose `tikv-cdc` is that:
1. I refer to this [PR](https://github.com/pingcap/tiup/pull/2017) that using `tidb-dashboard` & `tidb-dashboard_servers`.
2. The binary is named as `tikv-cdc`, so using `tikv_cdc` will be error-prone, e.g. `run_tikv-cdc.sh.tpl` will has both `tikv-cdc` & `tikv_cdc`.

Or how about using `kvcdc` ? Just as what we add to playground (#2000) ?

Reply:
I think #2017 should also change to use identical name formatting, I'm absolutely sure that mixing `-` and `_` in the same key name will cause chaos to the end users.

`kvcdc` would be a good choice IMO.

Reply:
Okay~ I will change to `kvcdc`.

Reply:
Done. PTAL~